### PR TITLE
Use labels in AlignedUMAP if provided

### PR DESCRIPTION
**Pull request by gclen**
Fix issue #801

### CODE REVIEW

**Comment by lmcinnes** [comment]
Looks like all the test failures are due to some tensorflow/keras changes that have broken parametricUMAP; I'll just merge this since it looks like a very simple change.
